### ISSUE TITLE
TEX Equation Numbering Only Works on New File

### ISSUE THREAD

**Issue by lightsquared** [nteract/archived-desktop-app#3645]
This odd behavior only occurs on nteract desktop for OSX (0.12.3).

If I create a new document (python 3 or Julia 1.0) and convert the first cell created to markdown I can then past the following code into the cell and I get equation numbering in the processed md cell:

`$$
\begin{equation}
a + b = c \tag{abc}\label{eq:one}
\end{equation} 
$$

I refer you to e\eqref{eq:one}.`

If you attempt to do this in an existing nteract document (python3) the code is not executed at all.  I would expect the equation numbering to work regardless of when the document was created.

![screen shot 2018-11-12 at 6 39 29 am](https://user-images.githubusercontent.com/9501140/48348162-1d641e80-e646-11e8-8349-7a09cc97de92.png)
![screen shot 2018-11-12 at 6 40 00 am](https://user-images.githubusercontent.com/9501140/48348178-281eb380-e646-11e8-9b24-2dddb6b632ac.png)

**Desktop (please complete the following information):**
 - OS: macOS Majove 12.14
 - Browser safari
 - Version 12

**Comment by willingc** [comment]
Thanks for the report @lightsquared.

**Comment by willingc** [comment]
Able to reproduce this behavior on macOS 10.14.1, nteract 0.12.3 for a new Python 3 notebook.
<img width="756" alt="screenshot 2018-12-12 20 51 42" src="https://user-images.githubusercontent.com/2680980/49916462-ccfffc80-fe4f-11e8-9315-efdf37e184c7.png">

**Comment by willingc** [comment]
Error before reloading kernel
<img width="1176" alt="screenshot 2018-12-12 20 55 01" src="https://user-images.githubusercontent.com/2680980/49916704-ff5e2980-fe50-11e8-8243-e78194c836ef.png">
After restarting kernel (resolved the above error)
<img width="644" alt="screenshot 2018-12-12 20 57 55" src="https://user-images.githubusercontent.com/2680980/49916705-ff5e2980-fe50-11e8-8b85-1793933a8c18.png">

**Comment by willingc** [comment]
This is likely related to MathJax and its handling of `\label{eq:one}` since removing this statement renders the equation. [ref](https://docs.mathjax.org/en/v2.5-latest/tex.html#)

I'm going to close this for now. If others have additional issues, please request us to reopen or file a new issue. Thanks.
